docs/ref/contrib/admin/index.txt

```diff
@@ -2717,3 +2717,13 @@ The action in the examples above match the last part of the URL names for
 :class:`ModelAdmin` instances described above. The ``opts`` variable can be any
 object which has an ``app_label`` and ``model_name`` attributes and is usually
 supplied by the admin views for the current model.
+
+.. currentmodule:: django.contrib.admin.views.decorators
+
+The staff_member_required decorator
+===================================
+
+.. function:: staff_member_required([redirect_field_name=REDIRECT_FIELD_NAME, login_url='admin:login'])
+    :noindex:
+
+    See :func:`~django.contrib.admin.views.decorators.staff_member_required` for usage notes.
```

Comment:
To get a sense for what info should be in the reference doc vs. a topic guide, compare ref/forms/models.txt and topics/forms/modelforms.txt.